Dockerfile

```diff
@@ -0,0 +1,161 @@
+# Copyright 2014 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+##########################################################################
+
+# INSTRUCTIONS:
+#
+# This Dockerfile allows for an easy installation of Oppia for Windows
+# users and a more reliable testing environment for running test scripts.
+# 
+# Current known limitations:
+#   - no support for run_e2e_tests.sh
+#   - run_frontend_tests.sh might not run correctly every time
+#
+##########################################################################
+###################### TO SET UP THE OPPIA REPOSITORY ####################
+##########################################################################
+#
+# STEP 1
+# Download Docker Desktop for Windows here:
+# https://hub.docker.com/editions/community/docker-ce-desktop-windows
+#
+# STEP 2
+# Start Docker by clicking on the Docker application (a Docker icon should
+# appear in your taskbar tray).
+#
+# STEP 3
+# Build the Docker image from the Oppia root folder (note the "." at the
+# end of the command):
+#
+#   docker build -t {some_name} .
```

Comment:
Please explain what "some_name" is. here and elsewhere

Reply:
Can do.